src/components/navigation-bar/navigation-bar.jsx

```diff
@@ -12,7 +12,7 @@ const NavigationBar = (props) => {
           {
             props.children.map((child, key) => {
               return (
-                <Tab key={key} eventKey={key} title={child.props.title} className="custom-tab-content">
+                <Tab mountOnEnter key={key} eventKey={key} title={child.props.title} className="custom-tab-content">
```

Comment:
Can we reformat this now that we've added another attribute?

```jsx
<Tab
  key={key} 
  eventKey={key} 
  title={child.props.title} 
  className="custom-tab-content"
  mountOnEnter={true}
>
  {child}
</Tab>
```

Reply:
Done 👍